pom.xml

```diff
@@ -20,7 +20,7 @@
         </repository>
         <repository>
             <id>bstats-repo</id>
-            <url>http://repo.bstats.org/content/repositories/releases/</url>
+            <url>https://mvnrepository.com/artifact/org.bstats/bstats-bukkit</url>
```

Comment:
If you look at the [bStats docs](https://bstats.org/getting-started/include-metrics), the correct repo URL is: https://repo.codemc.org/repository/maven-public

Let's use the correct repo rather than mvnrepository.com

Reply:
Interesting.